### PR TITLE
examples/gnrc_border_router: add commented SLIP compatibility

### PR DESCRIPTION
**Pull request by kYc0o**
A trial to solve #6354

### CODE REVIEW

**Comment by kYc0o** [comment]
README.md fixed.

File: examples/gnrc_border_router/README.md [hunk 1]
```diff
@@ -217,7 +217,7 @@ USEMODULE += gnrc_slip
 ## Configuration
 
 In order to connect a RIOT 6LoWPAN border router over SLIP you run a small
-program called tunslip (imported from Contiki) [2] on the Linux host. The
+program called tunslip6 (imported from Contiki) [2] on the Linux host. The
```
Comment (miri64) [inline examples/gnrc_border_router/README.md]: For ethos this is `dist/tools/ethos`, no?

Comment (kYc0o) [inline examples/gnrc_border_router/README.md]: Yes, it is.

Comment (miri64) [inline examples/gnrc_border_router/README.md]: I think the doc should reflect that ;-). Also that either tool is only needed if ethos or slip is used. For an ethernet device no such tool is needed.

Comment (kYc0o) [inline examples/gnrc_border_router/README.md]: I don't get what you mean. The instructions for ethos are given at the beginning of this readme. For ethernet I don't know if we're giving explicit instructions or just saying that's a possibility.

Should we give _also_ instructions for an ethernet device?

Comment (miri64) [inline examples/gnrc_border_router/README.md]: > The instructions for ethos are given at the beginning of this readme.

Arghs, sorry

> Should we give also instructions for an ethernet device?

Nope, people should know how to plug-in an ethernet cable and start uhcp ;-)

Comment (kYc0o) [inline examples/gnrc_border_router/README.md]: Great! Kein problem ;)

**Pull request merged**